.circleci/config.yml

```diff
@@ -115,8 +115,11 @@ jobs:
               cd example
               sed -i -- 's/npm/src/g' spec/config.yml
 
+              # deploy latest iam roles to aws
+              ./node_modules/.bin/kes cf deploy --kes-folder iam --region us-east-1 --deployment cumulus-from-source
```

Comment:
I say this is redundant. In the unlikely case of updating the iam, we just have to update it from a local machine.

Reply:
So the thing is, these tests will not work without updating the iam, which is why I put it in. So everyone will have to update their iams after I merge. Also I'd have to update the npm test stack (source was updated with this). Is it fine to leave it? Just seems easier to automatically grab any iam changes.

Reply:
sounds good. let's keep it here.